File: solutions/raindrops/Python/raindrops.py
def convert(number):
    s = ""

    fizzbuzz = dict({3: 'Pling', 5: 'Plang', 7: "Plong"})

    for n in fizzbuzz:
        if number % n == 0:
            s += fizzbuzz[n]
    if s == "":
        s = str(number)
    return s

File: solutions/raindrops/Erlang/src/raindrops.erl
-module(raindrops).
-export([convert/1]).

convert(Number) ->
Result = lists:concat([if Number rem 3 == 0 -> "Pling" ; true -> ""  end,
                       if Number rem 5 == 0 -> "Plang" ; true -> "" end,
                       if Number rem 7 == 0 -> "Plong" ; true -> ""  end
                       ]),
if Result == "" -> integer_to_list(Number);
    true -> Result
end.
